web/themes/custom/recipe_theme/templates/views/views-view-unformatted--catalog.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+<div class="row">
+
+{% for row in rows %}
+  {%
+    set row_classes = [
+      default_row_class ? '',
```

Comment:
This line says: if `default_row_class` is true, this string should be empty.

1. You either don't need this altogether, or
1. You should provide a default class if the option is enabled.

By the way, this value comes from here: https://github.com/Pronovix/food/pull/36/files#diff-e6f4d67045c52ba2f7c35c268cd6f31eR74